src/forms/AdminVaultForm.tsx

```diff
@@ -3,22 +3,28 @@ import { z } from 'zod';
 import { createForm } from './createForm';
 import { zEthAddress } from './formHelpers';
 
-const AssetEnum = z.enum([
+const zAssetEnum = z.enum([
   'DAI',
   'USDC',
   'YFI',
   'SUSHI',
-  'alUSD',
+  'ALUSD',
   'USDT',
+  'ETH',
   'OTHER',
 ]);
+export type AssetEnum = z.infer<typeof zAssetEnum>;
 // type TEpochRepeatEnum = typeof AssetEnum['_type'];
 
-const schema = z
+export const schema = z
   .object({
     token: z.number(),
-    asset: AssetEnum,
-    custom_asset: zEthAddress,
+    asset: zAssetEnum,
+    custom_asset: z.string().refine(async val => {
```

Comment:
Actually this isn't needed, as the CreateVaultForm now is for that one modal only.

The other modals might be able to share a form, I think we need design clarity